src/ConstantTask.java

```diff
@@ -0,0 +1,17 @@
+/*
+    Task: Create final static collection. Add and change elements.
+ */
+public class ConstantTask {
+
+    /*
+        There is no problem to add remove or update elements in static
+        final collection. If we try to change reference we will have error.
+        Collection is final and not immutable.
+     */
+    public static void main(String[] args) {
```

Comment:
the main purpose has reached! But general design of our library is not good. If we create two libraries and add book, it will appear in both of them. (may be libraries use common box with books).

Reply:
Yes I know. The main idea of the list BOOKS was to keep books, which are in each library and not for the unique books in them. Maybe not the best example.